app/models/message_draft.rb

```diff
@@ -2,4 +2,5 @@
 
 # Message drafts are essentially messages with an implied folder (drafts)
 class MessageDraft < Message
+  alias_attribute :draft_id, :id
```

Comment:
I don't understand why this is necessary. I think this makes things more confusing. Maybe we should have MessageDraft serializer so that the JSON is more clear that the object is a draft and not a Message, then you don't need to have this different id.